controllers/main.controllers.js

```diff
@@ -41,6 +41,18 @@ exports.deleteProject = async (req, res, next) => {
 // @route   DELETE api/client/:id
 exports.deleteClient = async (req, res, next) => {
   // Delete client with req.params.id
+  try {
+    const { id: _id } = req.params;
+    const docs = await Client.findByIdAndDelete(_id);
+    // console.log(docs);
+    if(docs != null) res.status(200).json(docs);
+    else {
+      res.status(200).json("No client found");
+    }
+  } catch (err) {
+    // console.log(err);
```

Comment:
for error response should be 
`{
success: false, error: {//err message goes here}
}`
send err.message string

Reply:
@VighneshManjrekar the requested changes have been made. Please review and verify if further changes are needed.